Add more styles and sample html

kalgynirae opened the pull request:
This commit adds two sample html files, login.html and list.html, and makes
some changes to demo.html. The global css file now contains styling for
buttons, forms, and a menu bar.

@PiPeep will be happy to know that the button hover color is not currently yellow, although I personally still like the yellow.

kalgynirae committed:
Add more styles and sample html

This commit adds two sample html files, login.html and list.html, and makes
some changes to demo.html. The global css file now contains styling for
buttons, forms, and a menu bar.

bgw commented on src/html/demo.html:
```diff
@@ -7,11 +7,18 @@
+      <h1 id="logo"><img src="img/routemaster-white.png" width="144" height="28" alt="RouteMaster"></h1>
+    </header>
     <nav>
+        <a id="go-track" href="">Track</a>
```
You indented too much here.

bgw commented on src/html/list.html:
```diff
@@ -0,0 +1,47 @@
+      <h1 id="logo"><img src="img/routemaster-white.png" width="144" height="28" alt="RouteMaster"></h1>
+    </header>
+    <nav>
+        <a id="go-track" href="">Track</a>
```
Again, you indented too much.

bgw commented on src/html/login.html:
```diff
@@ -0,0 +1,103 @@
+  <link href="css/global.css" rel="stylesheet" type="text/css">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <script type="text/javascript">
+    window.addEventListener('hashchange', function () {
```
I know this is just a demo and will change, but to give others an idea of how to do this correctly, go ahead an include zepto, and use the `$` selectors.

kalgynirae committed:
Fix indentation in html files

bgw commented on src/lcss/global.css:
```diff
@@ -1,21 +1,55 @@
-/* Grab fonts from Google */
-@import url('http://fonts.googleapis.com/css?family=Source+Sans+Pro:400,700,600');
-@import url('http://fonts.googleapis.com/css?family=Source+Code+Pro');
+/* Fonts */
```
LESS has [compile-time imports](http://lesscss.org/#-importing). I think these sections should be split into multiple files.

---

Edit: Also, the font blocks are 2-space indented, but the rest of the file is 4-space indented.

kalgynirae committed:
Use Zepto.js in the demo login page

kalgynirae committed:
Fix indentation and rename .css to .less

We've renamed src/lcss/*.css to src/less/*.less to better reflect that our
source files are LessCSS files. Maven runs them through the LessCSS compiler
and puts the output in bin/css/*.css.
